app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Okay, then I think that for now it is maybe better to keep them together. In future, when protocol stabilizes and we have a number of different implementations we'll have to keep and change them separately anyway. But since we still make many changes and develop the protocol intensively perhaps it's better to have them side by side here so it is easier for us to maintain. We can change this in future and move it out into a separate gem.